nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# Photo Upload

# Description
Can now Upload a file or use Image URL when adding a post or editing a post
Fixes # (38)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
``Git fetch origin ak-photo-upload``
``Git checkout origin ak-photo-upload``


When adding or editing a post, you can upload an image file(only file types .png, .bmp, .jpeg, .jpg, and .gif) or input a URL for an image. A preview of the image should display. When editing a post, no preview will display unless the image is displayed

# Checklist:
